docs/reference/storage.md

```diff
@@ -0,0 +1,127 @@
+# Storage
+
+uv persists data in several locations on your system.
+
+## Directory Strategies
+
+uv follows platform conventions for determining where to store different types of data.
+
+Generally, it's best to configure these rather than each uv-specific storage location.
+
+### Cache
+
+For temporary files and caches:
+
+- `$XDG_CACHE_HOME/uv` or `$HOME/.cache/uv` on Unix systems
+- `%LOCALAPPDATA%\uv\cache` on Windows
+
+### Data
+
+For persistent application data:
+
+- `$XDG_DATA_HOME/uv` or `~/.local/share/uv` on Unix systems
+- `%APPDATA%\uv\data` on Windows
+- `.uv` in the working directory as a fallback
+
+### Config
+
+For user configuration files:
+
+- `$XDG_CONFIG_HOME/uv` or `~/.config/uv` on Unix systems
+- `%APPDATA%\uv` on Windows
+
+For system configuration files:
+
+- `$XDG_CONFIG_DIRS/uv` or `/etc/uv` on Unix systems
+- `%PROGRAMDATA%\uv` on Windows
+
+### Binaries
+
+For executable files:
+
+- `$XDG_BIN_HOME`, `$XDG_DATA_HOME/../bin`, or `~/.local/bin` on all platforms
+
+## Cache
+
+uv uses a local cache to avoid re-downloading and re-building dependencies. The cache contains
+wheels, source distributions, responses from package indexes, Git repositories, and Python
+interpreter metadata.
```

Comment:
Can we add a note that the cache needs to be on the same filesystem as the venv for best performance?

Reply:
That seems like a `cache.md` thing, but yes.